signalfx/resource_signalfx_dashboard.go

```diff
@@ -36,6 +36,12 @@ func dashboardResource() *schema.Resource {
 				Optional:    true,
 				Description: "Description of the dashboard (Optional)",
 			},
+			"tags": &schema.Schema{
+				Type:        schema.TypeList,
```

Comment:
Should this be a `schema.TypeSet`?

The only issue with making it a `schema.TypeSet` is I am not sure if I can check the values of the tags as done [here](https://github.com/splunk-terraform/terraform-provider-signalfx/pull/349/files#diff-b913ab62f26f9f3a350f1d6fffce493edf1d232cca5362e70c4a58161f564467R90).

Reply:
since the api don't restrict tags to be unique, I think List is the right type here.